### PR TITLE
gismo 24.08.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by p-linnane** [comment]
Not the latest release: https://github.com/gismo/gismo/releases

**Comment by chenrui333** [comment]
https://github.com/gismo/gismo/issues/728

**Comment by chenrui333** [comment]
close for now.

**Comment by chenrui333** [comment]
now it is the latest release

image

File: Formula/g/gismo.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Gismo < Formula
   desc "C++ library for isogeometric analysis (IGA)"
   homepage "https://gismo.github.io"
-  url "https://github.com/gismo/gismo/archive/refs/tags/v23.12.0.tar.gz"
-  sha256 "6dc78e1d0016a45aee879eec0e42faf010cd222800461d645f877ff0c1f2d1a2"
+  url "https://github.com/gismo/gismo/archive/refs/tags/v24.08.0.tar.gz"
+  sha256 "e72ca05e6a836748a04db5006e7fdf576fc22f0d6e369b2209875d95c17499fe"
```
Comment (chenrui333) [inline Formula/g/gismo.rb:5]:
```suggestion
  sha256 "ac6e7fc9d40aae698f3451a62dbbe45d9c62a40dfd1caf690b4d10eb624bcd6a"
```

**Review by carlocab — APPROVED** [review]

**Pull request merged**